test/test.mli

```diff
@@ -1,11 +1,13 @@
 (*Ideally we should have a unified interface of testing framework, I am still thinking about how to abstract away the difference between JS testing and OUnit.*)
```

Comment:
When I started this there is no `bs-jest`. Let's experiment with `ava` here and move the interface to `bs-ava` later when we ready to ship?